common/djangoapps/student/views.py

```diff
@@ -2497,7 +2497,7 @@ def _get_course_programs(user, user_enrolled_courses):  # pylint: disable=invali
 
     for course_key, programs in course_programs.viewitems():
         for program in programs:
-            if program.get('status') == 'active' and program.get('category') == 'xseries':
+            if program.get('status') == 'active' and program.get('category') == 'XSeries':
```

Comment:
This means that Micromaster programs do not include the object below this if statement. Is this correct? Can you help explain?